datalad/plugin/wtf.py

```diff
@@ -187,7 +201,8 @@ def _t2s(t):
                      if ds_meta else '[no metadata]'
             ),
             externals=external_versions.dumps(preamble=None, indent='', query=True),
-            metaextractors='\n'.join(p for p in dir(metaextractors) if not p.startswith('_')),
+            extensions='\n'.join(ep.name for ep in iter_entry_points('datalad.extensions')),
+            metaextractors='\n'.join('{}: {}'.format(k, v) for k, v in extractors.items()),
```

Comment:
use `format_dict(extractors)` for uniform appearance/handling?

Reply:
Done